src/Raven.Server/Documents/Indexes/Static/CurrentIndexingScope.cs

```diff
@@ -36,6 +36,16 @@ public class CurrentIndexingScope : IDisposable
         /// [collection: [key: [referenceKeys]]]
         public Dictionary<string, Dictionary<Slice, HashSet<Slice>>> ReferencesByCollectionForCompareExchange;
 
+        /*
+        public Dictionary<string, Dictionary<string, LoadFailure>> MismatchedReferences;
+
+        private const int MaxMismatchedReferencesPerSource = 10;
+        private const int MaxMismatchedDocumentLoadsPerIndex = 10;
+        
+        private bool _lastLoadMismatched = false;
+        */
+        internal MismatchedReferencesHandler _mismatchedReferencesHandler = new MismatchedReferencesHandler();
```

Comment:
_mismatchedReferencesHandler -> MismatchedReferencesHandler since it's used externally and it looks awkward there

Reply:
Done